edsdme/blocks/partners-navigation/features/profile/dropdown.css

```diff
@@ -24,6 +24,13 @@
   transform: translate3d(0,0,0); /* Fix Safari issues w/ position: sticky */
 }
 
+/*MWPW-158926*/
+.feds-profile-menu {
```

Comment:
Solution is good, but I think we should keep the way we write the css selectors consistent
`.feds-profile-menu .company-placeholder `